src/texify.js

```diff
@@ -109,8 +111,13 @@ TeXify.prototype.renderTexFile = function (file) {
                         console.log(stdout)
 
                         try {
-                            let svgBaseUrl = urljoin(this.push.repository.html_url, '/master/').replace('github.com', 'rawgit.com')
-                            fs.writeFileSync(tmpOutputPath, fs.readFileSync(tmpOutputPath, 'utf8').replace(new RegExp(this.treeLocalPath, 'g'), svgBaseUrl))
+                            let svgBaseUrl = urljoin(this.push.repository.html_url, '/master/').replace('://github.com', '://raw.githubusercontent.com')
+
+                            let tmpOutputContents = fs.readFileSync(tmpOutputPath, 'utf8')
+                                .replace(new RegExp(this.treeLocalPath, 'g'), svgBaseUrl)
```

Comment:
What's the point of doing this at all? Do relative links not work? This is enforcing using the master branch's version of the rendered tex, which may not exist

Reply:
You're totally right. We should use relative urls. Gonna open a new issue